.nvmrc

```diff
@@ -1 +1 @@
-8
\ No newline at end of file
+14
```

Comment:
This was a change @ggdaltoso and I previously discussed where the Node version here wasn't compatible with the yarn lock file